stripe/src/main/java/com/stripe/android/model/Address.java

```diff
@@ -16,7 +19,18 @@
  * Model for an owner <a href="https://stripe.com/docs/api#source_object-owner-address">address</a>
  * object in the Source api.
  */
-public class SourceAddress extends StripeJsonModel {
+public class Address extends StripeJsonModel {
+
+    @IntDef({
+            RequiredAddressFields.NONE,
+            RequiredAddressFields.ZIP,
+            RequiredAddressFields.FULL})
+    @Retention(RetentionPolicy.SOURCE)
+    public @interface RequiredAddressFields {
```

Comment:
I think it's worth calling this `RequiredBillingAddressFields`, since we'll want to allow a larger set of options for shipping (on iOS we allow specifying the individual fields you want in your shipping form).

wdyt @bdorfman-stripe ?